src/v/kafka/protocol/schemata/generator.py

```diff
@@ -1080,14 +1080,6 @@ class response;
 {%- set decoder, named_type = field.decoder(flex) %}
 {%- if named_type == None %}
     return reader.{{ decoder }};
-{%- elif field.nullable() %}
```

Comment:
what makes this case specific to arrays of integers? which case handles nullable fields in normal array-of-structs?